LKF Teamsmeeting (#74)
importPI19fromDHGE/dhge-pi19-sem4: SWE-KASCHE/appendix/LKF.md

- ToDo

> **Zum Zentralen System**

Reviewer: ```suggestion
**Zum zentralen System**
```

- keine Intelligenz bei Terminals
- Terminal 1, Terminal 2, Terminal 3 ... Terminal N $\rightarrow$ greifen alle auf das zentrale System zu

> - typische Vor-und- Nachteile eines zentralen Systems

Reviewer: ```suggestion
- typische Vor- und Nachteile eines zentralen Systems
```

> - gemeinsamer Datenbestand = einfach einzurichten

Reviewer 2: ```suggestion
- gemeinsamer Datenbestand $\rightarrow$ einfach einzurichten
```

Reviewer: ```suggestion
  - gemeinsamer Datenbestand = einfach einzurichten
```

> - Last kann System in die Knie zwingen
> - gemeinsame Nutzung von gleicher Ressource: Kollision muss behandelt oder vermieden werden, v.a. bei Datenbeständen!

Reviewer: ```suggestion
  - Last kann System in die Knie zwingen
  - gemeinsame Nutzung gleicher Ressourcen: Kollision muss behandelt oder vermieden werden, v.a. bei Datenbeständen!
```
